packages/dashboard/src/pages/segments/index.page.tsx

```diff
@@ -49,9 +43,52 @@ export const getServerSideProps: GetServerSideProps<PropsWithInitialState> =
       }
       return result.value;
     });
+    const journeys = await prisma().journey.findMany({
+      where: {
+        workspaceId,
+      },
+    });
+    const computedPropertyPeriods =
+      await prisma().computedPropertyPeriod.findMany({
```

Comment:
Can you re-use this method https://github.com/dittofeed/dittofeed/blob/be8609a526c478e065efdf1faa7fdfe5fe10f37c/packages/backend-lib/src/computedProperties/computePropertiesIncremental.ts#L240C16-L240C46 instead of doing a simple `findMany`?

Reply:
Ok sure. Just a small doubt, I'm not sure how to provide the `step` value. Should I directly mention the step like this? 
```ts
const computedPropertyPeriods = await getPeriodsByComputedPropertyId({
  workspaceId,
  step: ComputedPropertyStep.ComputeState,
});
```
Or should I do something else?

Reply:
use the `ProcessAssignments` step!

Reply:
Ok sure!